src/js/components/accounts/confirm-email-verification.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import Radium from 'radium'
+import {RaisedButton} from 'material-ui'
+import {proxy} from 'settings'
+import SnackbarActions from 'actions/snackbar'
+
+let ConfirmEmailVerification = React.createClass({
+
+  contextTypes: {
+    router: React.PropTypes.object
+  },
+
+  propTypes: {
+    params: React.PropTypes.string
+  },
+
+  getStyles() {
+    let styles = {
+      container: {
+        textAlign: 'center',
+        background: '#f8f9fb',
+        height: '100%',
+        overflowY: 'auto'
+      },
+      content: {
+        width: '300px',
+        maxWidth: '90%',
+        padding: '20px',
+        margin: '0 auto 20px auto',
+        boxSizing: 'border-box'
+      },
+      button: {
+        width: '100%'
+      }
+    }
+    return styles
+  },
+
+  activateEmailAccount() {
```

Comment:
As @Pagebakers mentioned, we should try to keep the http requests outside of stores and rather outsource them to agents. We can keep this here for now, but once #415 is merged into develop and we have the appropriate http / ldp agent, this will have to be moved.